### DISCUSSION TITLE
希望可用id方式跳转漫画 以及翻页的问题

### DISCUSSION THREAD

**Discussion by Arunmor1510** [xiaoyaocz/flutter_dmzj#121] [Q&A] [answered]
1.  #57 给旧版引入了可以直接按id跳转的功能，希望新版能加回来
2.  似乎不管滑动方向设置为从左到右还是从右到左，手动点击漫画页的行为都是左边前往上一页，右边下一页

**Comment by xiaoyaocz** [comment, marked as answer]
1. 搜索框输入 id:\\{id}，如id:\4 跳转至漫画
2. #93 可开启左手模式
